packages/warriorjs-cli/src/ui/printWarriorStatus.js

```diff
@@ -10,10 +10,15 @@ import printLine from './printLine';
  */
 function printWarriorStatus(warrior) {
   const [screenWidth] = getScreenSize();
-  const warriorStatus = chalk.red(
+  const warriorStatus = chalk.redBright(
```

Comment:
nit: for the sake of correctness, we should rename this variable to `warriorHealth`.